<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KCrypt - A Luks Frontend</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/icon.png</normaloff>../icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="5">
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>562</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>110</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Volume</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Device</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mount Point</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Encryption</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mode</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Filesystem</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="mountButton">
      <property name="statusTip">
       <string/>
      </property>
      <property name="text">
       <string>Mount</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="dismountButton">
      <property name="statusTip">
       <string/>
      </property>
      <property name="text">
       <string>Dismount</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="createButton">
      <property name="statusTip">
       <string/>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>295</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Close</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVolumes">
    <property name="title">
     <string>&amp;Volumes</string>
    </property>
    <addaction name="actionMount"/>
    <addaction name="actionDismount_Luks_Volume"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_New_Luks_Volume"/>
    <addaction name="actionChange_Password"/>
    <addaction name="separator"/>
    <addaction name="actionBackup_Restore_Header"/>
   </widget>
   <widget class="QMenu" name="menuKeyfiles">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Keyfiles</string>
    </property>
    <addaction name="actionChange_Keyfile"/>
    <addaction name="actionCreate_Random_Keyfile"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuMisc">
    <property name="title">
     <string>&amp;Misc</string>
    </property>
    <addaction name="actionCreate_Random_Password"/>
    <addaction name="actionSecure_Erase"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionLanguage"/>
    <addaction name="actionShow_SystemTrayIcon"/>
    <addaction name="actionDelete_Mountpoint"/>
   </widget>
   <addaction name="menuVolumes"/>
   <addaction name="menuKeyfiles"/>
   <addaction name="menuMisc"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMount">
   <property name="text">
    <string>Mount Luks Volume</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionDismount_Luks_Volume">
   <property name="text">
    <string>Dismount Luks Volume</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionCreate_New_Luks_Volume">
   <property name="text">
    <string>Create New Luks Volume</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Change Luks Volume Password</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionErase_Password">
   <property name="text">
    <string>Delete Luks Volume Password</string>
   </property>
  </action>
  <action name="actionChange_Keyfile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add/Remove Keyfile to/from a Luks Volume</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionCreate_Random_Keyfile">
   <property name="text">
    <string>Create a random Keyfile</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionLuks_Volume_Info">
   <property name="text">
    <string>Luks Volume Info</string>
   </property>
  </action>
  <action name="actionSecure_Erase">
   <property name="text">
    <string>Secure Erase</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionBackup_Restore_Header">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Backup/Restore Volume Header</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionCreate_Random_Password">
   <property name="text">
    <string>Create Random Password</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionCheck_for_update">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Check for update</string>
   </property>
  </action>
  <action name="actionCreate_Container_File">
   <property name="text">
    <string>Create LUKS Container</string>
   </property>
  </action>
  <action name="actionShow_SystemTrayIcon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Systemtray Icon</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionDelete_Mountpoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Delete Mount Point</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
